README.md

```diff
@@ -22,6 +27,25 @@ Detailed instructions on how to implement your own integration with the mParticl
 Questions? Give us a shout at <support@mparticle.com>
 
 
+
```

Comment:
The installation instructions for kits are described in the core mParticle Apple SDK repository. Can we remove this from here?